js-foundations.js

```diff
@@ -3,27 +3,81 @@
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
 
+var test1 = 15
+var test2 = 0
+var test3 = 385
+var test4 = -7
+
+
+const isDivisible = (number) => {
+    if(number%3 === 0){
+        return `${number} is divisble by 3!`
+    } else {
+        return `${number} is not divisible by 3!`
+    }
+}
+
+console.log(isDivisible(Math.abs(test1)));
+console.log(isDivisible(Math.abs(test2)));
+console.log(isDivisible(Math.abs(test3)));
+console.log(isDivisible(Math.abs(test4)));
 
```

Comment:
To make your code a little cleaner, use the Math.abs in your function rather than on the call.